libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -125,23 +125,21 @@ template moveToMeshHelper(g: GossipSub,
                            peers = peerIds.len
 
     table[topic].excl(id) # always exclude
-    if id in g.mesh[topic]:
+    if id in g.mesh.getOrDefault(topic):
```

Comment:
hashset requires that init be called, so `getOrDefault` is not correct

Reply:
Not anymore apparently - https://github.com/status-im/Nim/blob/nimbus/lib/pure/collections/sets.nim#L127-L128.

Reply:
huh, there's another comment saying they are - oh well, maybe it works then.

Reply:
needs INIT, docs are wrong